assignments/objects.js

```diff
@@ -18,31 +18,62 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
+const interns = [
+  {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+  },
+  {
+    "id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M"
+  },
+  {
+    "id": 3,
+    "name": "Keven",
+    "email": "kmummery2@wikimedia.org",
+    "gender": "M"
+  },
+  {
+    "id": 4,
+    "name": "Gannie",
+    "email": "gmartinson3@illinois.edu",
+    "gender": "M"
+  },
+  {
+    "id": 5,
+    "name": "Antonietta",
+    "email": "adaine5@samsung.com",
+    "gender": "F"
+  },
+];
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(interns[0]["name"]);
 // Kennan's ID
-
+console.log(interns[1]["id"]);
 // Keven's email
-
+console.log(interns[2]["email"]);
 // Gannie's name
-
+console.log(interns[3]["name"]);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(interns[4]["gender"]);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
-
+console.log(kennan.speak());
```

Comment:
- Where is the kennan object that you are trying to access? How did you access the previous interns? Why are you changing it for kennan?
- Does the kennan object have a speak method (function) inside of it?